servers/rendering/rasterizer_dummy.h

```diff
@@ -186,7 +186,7 @@ class RasterizerSceneDummy : public RendererSceneRender {
 	void set_debug_draw_mode(RS::ViewportDebugDraw p_debug_draw) override {}
 
 	RID render_buffers_create() override { return RID(); }
-	void render_buffers_configure(RID p_render_buffers, RID p_render_target, int p_width, int p_height, RS::ViewportMSAA p_msaa, RS::ViewportScreenSpaceAA p_screen_space_aa, bool p_use_debanding, uint32_t p_view_count) override {}
+	void render_buffers_configure(RID p_render_buffers, RID p_render_target, int p_width, int p_height, bool p_is_scaled, RS::ViewportMSAA p_msaa, RS::ViewportScreenSpaceAA p_screen_space_aa, bool p_use_debanding, uint32_t p_view_count) override {}
```

Comment:
Instead of passing p_is_scaled  here, just check where relevant if the render target size is different than the render buffer size.

Reply:
That doesn't work because I don't have access to the original size in places like here:
https://github.com/BastiaanOlij/godot/blob/half_resolution_3d/servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp#L513

Reply:
Or do you mean setting `is_scaled` inside of `render_buffers_configure` by querying the size of `p_render_target` ?